feder/monitorings/templates/monitorings/_tabs.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n guardian_tags %}
+{% get_obj_perms request.user for object as "monitoring_perms" %}
```

Comment:
Zastanawiam się czy to nie spowoduje zbędnych zapytań, czy nie powinno się zmiennej ``monitoring_perms`` przekazywać do wstawki niż w niej to generować.
``{% get_obj_perms request.user for object as "monitoring_perms" %}`` generalnie odpowiada za pobranie danych z bazy danych, więc unikałbym powielania tego kodu.

Reply:
Też się nad tym zastanawiałem i wyszło mi, że nie, ponieważ:
1. szablon _tabs jest wstawiony tylko raz w stronie, za to w trzech stronach, więc chyba lepiej mieć to w jednym miejscu
2.  Nawet, gdyby nie, to bardzo proste i cachowalne zapytanie

Reply:
1/ Szablon ``_tabs`` jest wstawiony raz, ale ``get_obj_perms`` jest wstawiony dwa razy, bo drugi raz w przyciskach menu nawigacyjnego. Sprawdzę to sam...
3/ Zapytanie jest cachowalne, ale w Django cache działa przede wszystkim na poziomie obiektów (poza wprost zdefiniowanym cache), a więc jest 
3/ Można myśleć o cache bazy danych, ale jednak nie uważam za zasadne aż takie poleganie na tym, aby generować całkiem zbędne zapytania.

Zweryfikuje punkt 1, bo faktycznie gdzieś to się może cachować mądrze. Najprościej sprawdzić to przez ``python manage.py runserver_plus --print-sql``.

Reply:
sprawdziłem, zapytanie jest wykonywane tylko raz 
![image](https://user-images.githubusercontent.com/1237985/30502606-4bbb4578-9a67-11e7-86ca-f1047b061e5b.png)